include/circt/Dialect/FIRRTL/NLATable.h

```diff
@@ -65,6 +70,14 @@ class NLATable {
   // NLA tracking.
   void renameModule(StringAttr oldModName, StringAttr newModName);
 
+  // Rename the module names and also update the inner_sym, if the updated
+  // module name is at the leaf. renameMap is a map of old inner_sym names to
+  // new inner_sym names. Also set the updatedNLAs, if it is passed as argument.
+  void renameModuleAndInnerRef(
+      StringAttr newModName, StringAttr oldModName,
+      const DenseMap<StringAttr, StringAttr> &renameMap,
+      llvm::Optional<SmallVector<NonLocalAnchor> *> updatedNLAs = None);
```

Comment:
An optional pointer?  Isn't that just a pointer?

Reply:
It was redundant, removed the argument.